### PR TITLE
Add URL_SAFE_ALPHABET and PADDED_URL_SAFE_ALPHABET

### PR DESCRIPTION
**Pull request by benjaminoakes**

### CODE REVIEW

File: lib/base64.rb [hunk 1]
```diff
@@ -22,6 +22,11 @@
 # binary data into purely printable characters.
 
 module Base64
+  # The alphabet described in RFC 4648 and implemented by
+  # +urlsafe_encode64+ and +urlsafe_decode64+.
+  URL_SAFE_ALPHABET = "-0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ_abcdefghijklmnopqrstuvwxyz"
+  PADDED_URL_SAFE_ALPHABET = "-0123456789=ABCDEFGHIJKLMNOPQRSTUVWXYZ_abcdefghijklmnopqrstuvwxyz"
```
Comment (benjaminoakes) [inline lib/base64.rb:28]: These characters are listed in ASCII order

**Comment by benjaminoakes** [comment]
Replaced by https://github.com/ruby/ruby/pull/7572

**Comment by nobu** [comment]
Why not push another commit to the same branch?

**Comment by benjaminoakes** [comment]
Sorry about that.

I renamed the branch in GitHub and it closed the pull request.  Not intentional.

**Pull request closed without merging**